.jenkins/pytorch/common_utils.sh

```diff
@@ -66,7 +66,7 @@ function get_bazel() {
   chmod +x tools/bazel
 }
 
-TORCHVISION_COMMIT=e70c91a9ff9b8a20e05c133aec6ec3ed538c32fb
+TORCHVISION_COMMIT=883f1fb01a8ba0e1b1cdc16c16f2e6e0ef87e3bd
```

Comment:
This doesn't just update torchvision for docs, right? It'll also update whatever torchvision tests we have in CI, which is probably(?) fine as long as the tests pass

Reply:
Yeah correct. Also double-checked w/ Eli and doesn't appear to be any other CI tests that are building vision for docs :( Have this PR out to add vision back in: https://github.com/pytorch/pytorch/pull/46882/files